modules/vulnerability-management20-creating-report.adoc

```diff
@@ -12,18 +12,34 @@
 .Procedure
 . In the {product-title-short} portal, click *Vulnerability Management* -> *Vulnerability Reporting*.
 . Click *Create report*.
-. Enter a name for your report configuration in the *Report name* field.
-. Optional: Enter text describing the report configuration in the *Report description* field.
-. In the *CVE severity* field, select the severity of common vulnerabilities and exposures (CVEs) that you want to include in the report configuration.
-. Select the *CVE status*. You can select *Fixable*, *Unfixable*, or both.
-. In the *Image type* field, select whether you want to include CVEs from deployed images, watched images, or both.
-. In the *CVEs discovered since* field, select the time period for which you want CVEs to be included in the report configuration.
-. In the *Configure collection included* field, you must configure at least one collection. Complete any of the following actions:
-* Select an existing collection to include. To view the collection information, edit the collection, and get a preview of collection results, click *View*. When viewing the collection, entering text in the field searches for collections matching that text string.
-* Click *Create collection* to create a new collection.
+. In the *Configure report parameters* page, provide the following information:
```

Comment:
Consider a table for these entries.

Reply:
I would prefer to leave it as it is because typically we do not want to use tables in procedures.